### ISSUE TITLE
Used space is more than 100% of a backend

### ISSUE THREAD

**Issue by wimpers** [openvstorage/framework-alba-plugin#201]
![2016-09-12_8-59-25](https://cloud.githubusercontent.com/assets/8392927/18427278/64337994-78c7-11e6-894f-7a76fe547f98.jpg)

**Comment by khenderick** [comment]
Where does the framework get the data from:
* foreach namespace in ```show-namespaces```, sum up their ```storage``` value (let's call that sum ```ns_used```)
* foreach asd statistic in ```asd-multistatistics``` (from all by the framework claimed osds), sum up their ```capacity``` (let's call that sum ```capacity```) and ```disk_usage``` (let's call that sum ```used```)

In the above diagram:
* Free space = ```capacity``` - ```used```
* Used space = ```ns_used```
* Overhead = max(0, ```capacity``` - ```ns_used```)

As far as I remember from chats with @domsj, the ```ns_used``` might be too large because it takes the ideal written out data into account. E.g. it takes fragments into account that are not (yet) written. But I'm not sure whether this is still the case.

@wimpers, as always, feel free to specify which numbers to show.

**Comment by wimpers** [comment]
> feel free to specify which numbers to show.
The right ones.

What I assume is the issue is that fro a backend that is also used for the fragment cache, data which was removed from the cache isn't taken into account.

**Comment by domsj** [comment]
```
As far as I remember from chats with @domsj, the ns_used might be too large because it takes the ideal written out data into account. E.g. it takes fragments into account that are not (yet) written. But I'm not sure whether this is still the case.
```

That is indeed still the case. I suggest summing the sizes of the storage per osd to get the desired result

**Comment by khenderick** [comment]
So, let's conclude I use - in the above example - ```capacity``` and ```used``` to display everything, and get rid of ```overhead```. Can we agree on that, @wimpers?

**Comment by wimpers** [comment]
For used, use storage per OSD and remove overhead.

**Comment by khenderick** [comment]
Fixed by #279, packaged in openvstorage-backend-1.7.6-rev.822.ff4199c

**Comment by JeffreyDevloo** [comment]
Closing. 
Should be reopened if we ever face this issue again.
